packages/components/src/Menu/Menu.tsx

```diff
@@ -22,22 +21,24 @@ export const Menu: React.FC<MenuProps> = ({
   children,
   ...props
 }) => {
-  const itemStyles = useStyles({
-    css: {
-      position: 'absolute',
-      minWidth: '120px',
-      display: 'block',
-      textAlign: 'left',
-      borderRadius: '2px',
-    },
-  });
-
   return (
     <Box variant={`menu.${variant}`} {...props}>
       <Button onClick={onClick} variant="menu">
         {label}
       </Button>
-      {show ? <div className={itemStyles}>{children}</div> : null}
+      {show ? (
+        <Box
+          display="block"
+          position="absolute"
+          minWidth="120px"
+          borderRadius="2px"
+          css={{
+            textAlign: 'left',
```

Comment:
I don't think that's needed, is it? It actually excludes an optional `text-align: right` or `center`.